src/app/pages/BibPage.jsx

```diff
@@ -62,7 +61,7 @@ export const BibPage = (
   const checkForMoreItems = useCallback((once = false) => {
     if (!bib || !bib.items || !bib.items.length || (bib && bib.done)) {
       // nothing to do
-    } else if (bib && bib.items.length >= numItemsTotal) {
+    } else if (bib && bib.items.length >= numMatched) {
```

Comment:
Typo: should be `numItemsMatched`.

Similar comment to above: I set this
```
const numToCheck = numItemsMatched ? numItemsMatched : numItemsTotal;

....
 } else if (bib && bib.items.length >= numToCheck) {
```

because I thought `numItemsMatched` could be 0.